RuntimeDetour/Platforms/Runtime/DetourRuntimeNETCorePlatform.cs

```diff
@@ -1,24 +1,158 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Reflection.Emit;
-using MonoMod.Utils;
-using System.Linq;
-
-namespace MonoMod.RuntimeDetour.Platforms {
-#if !MONOMOD_INTERNAL
-    public
-#endif
-    class DetourRuntimeNETCorePlatform : DetourRuntimeNETPlatform {
-
-        protected override void DisableInlining(MethodBase method, RuntimeMethodHandle handle) {
-            // https://github.com/dotnet/runtime/blob/89965be3ad2be404dc82bd9e688d5dd2a04bcb5f/src/coreclr/src/vm/method.hpp#L178
-            // mdcNotInline = 0x2000
-            // References to RuntimeMethodHandle (CORINFO_METHOD_HANDLE) pointing to MethodDesc
-            // can be traced as far back as https://ntcore.com/files/netint_injection.htm
-
-            // FIXME: Take a very educated guess regarding the offset to m_wFlags in MethodDesc.
-        }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Reflection.Emit;
+using MonoMod.Utils;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System.IO;
+using MonoMod.Common.RuntimeDetour.Platforms.Runtime;
+
+namespace MonoMod.RuntimeDetour.Platforms {
+#if !MONOMOD_INTERNAL
+    public
+#endif
+    class DetourRuntimeNETCorePlatform : DetourRuntimeNETPlatform {
+

```

Comment:
Are you committing with different line endings? I'll definitely need to fix this up after merging 😅

Reply:
Yeah, it seems like my VS setup likes inserting CRLF for new lines, and it causes inconsistencies. I usually use CRLF, and so just told it to fix them all to that, but I'll go ahead and make sure that the line endings are LF from here on.